src/js/brutusin-json-forms.js

```diff
@@ -1311,7 +1307,7 @@ if (typeof brutusin === "undefined") {
 
         function cleanSchemaMap(schemaId) {
             for (var prop in schemaMap) {
-                if (prop.startsWith(schemaId)) {
+                if (prop.startsWith(schemaId)) { // 02_AddLevel1Item_Delete_AgainAdd.png
                     delete schemaMap[prop];
                 }
             }

```

Comment:
Surprisingly, in the origin itself, the code differs between `src` and `dist`!!!

`src` has this fix... but `dist` doesn't have the fix.